haiku-apps/clipdinger/clipdinger-0.2.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="A tool to manage a history of the system clipboard" 
+DESCRIPTION="Clipdinger solves the problem that you often have to paste some \
+text you've just recently copied to the clipboard, but which has been \
+replaced by something you've copied more recently... It also saves the \
+history so it'll appear just as you left it on the last shutdown.
+
+Clipdinger provides a history of your recent clippings and should be started \
+automatically on every boot-up by creating a link to it at ~/config/settings/\
+boot/launch/.
+
+You access the history by pressing SHIFT + ALT + V, which opens the \
+Clipdinger window where you can select an entry with the cursor keys and \
+paste it by hitting RETURN. ESCAPE minimizes the window without pasting."
+HOMEPAGE="http://humdingerb.github.io/clipdinger/" 
+SOURCE_URI="git://github.com/humdingerb/clipdinger.git#a57bce02c91462a30239d32373d4421e70bc5686"
+LICENSE="MIT"
+COPYRIGHT="2010-2015 Humdinger"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	clipdinger = $portVersion
+	app:Clipdinger = $portVersion
+	add_on:ClipdingerInputDevice = $portVersion
+	add_on:ClipdingerInputFilter = $portVersion
+	"
+REQUIRES="haiku"
+
+BUILD_REQUIRES="haiku_devel"
+
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+USER_SETTINGS_FILES="
+	settings/Clipdinger directory "keep-old"
```

Comment:
Quotes here are unnecessary (and due to how Bash works, they're stripped out anyway).